tests/integration/dart/test/credential_issuance_client_test.dart

```diff
@@ -23,119 +24,106 @@ void main() {
         passphrase: env.passphrase,
       );
 
-      // issuance client
       final issuanceClient = AffinidiTdkCredentialIssuanceClient(
-          dio: Dio(BaseOptions(
-            baseUrl: AffinidiTdkCredentialIssuanceClient.basePath,
-            connectTimeout: const Duration(seconds: 5),
-            receiveTimeout: const Duration(seconds: 5),
-          )),
-          authTokenHook: authProvider.fetchProjectScopedToken);
-      configurationApi = issuanceClient.getConfigurationApi();
+        dio: Dio(BaseOptions(
+          baseUrl: AffinidiTdkCredentialIssuanceClient.basePath,
+          connectTimeout: const Duration(seconds: 5),
+          receiveTimeout: const Duration(seconds: 5),
+        )),
+        authTokenHook: authProvider.fetchProjectScopedToken);
 
-      // wallet client
-      final walletClient = AffinidiTdkWalletsClient(
-          dio: Dio(BaseOptions(
-            baseUrl: AffinidiTdkWalletsClient.basePath,
-            connectTimeout: const Duration(seconds: 5),
-            receiveTimeout: const Duration(seconds: 5),
-          )),
-          authTokenHook: authProvider.fetchProjectScopedToken);
-      walletApi = walletClient.getWalletApi();
-
-      // Create wallet
-      final didKeyInputBuilder = DidKeyInputParamsBuilder()
-        ..name = 'Test Wallet';
-      final walletInputBuilder = CreateWalletInputBuilder()
-        ..oneOf = OneOf2<DidKeyInputParams, DidWebInputParams>(
-            value: didKeyInputBuilder.build(), typeIndex: 0);
-      final createdWallet = (await walletApi.createWallet(
-              createWalletInput: walletInputBuilder.build()))
-          .data;
-      walletId = createdWallet!.wallet!.id!;
+      configurationApi = issuanceClient.getConfigurationApi();
     });
 
-    tearDown(() async {
-      if (walletId.isNotEmpty) {
-        await walletApi.deleteWallet(walletId: walletId);
+    test('Deletes issuance configuration if exists', () async {
```

Comment:
Let's make sure delete always runs, and not skip it conditionally. On `setUp` and `cleanUp` we can create and delete configs if necessary.

Reply:
Should we delete all wallets?
If so, then we could delete undesired wallets.

Reply:
No, I meant the configuration. If there is none we are not testing deletion of it.

Reply:
Please give your view on how Projects and Issuance configs should be tested. 
There are system restrictions that limits CRUD testing of those.

F.e., we cannot delete projects, and there is a limit how many of those per account. 
Also, there could be only 1 issuance configuration per account. Which means, that our CRUD integration test will delete existing config. That might be fine for the pipeline (for our account), but for someone to run this locally, vs his/her account, the result might be a surprise.